src/Compilers/CSharp/Portable/Lowering/SyntheticBoundNodeFactory.cs

```diff
@@ -612,6 +612,16 @@ public BoundCall Call(BoundExpression receiver, MethodSymbol method, ImmutableAr
             { WasCompilerGenerated = true };
         }
 
+        public BoundCall Call(BoundExpression receiver, MethodSymbol method, ImmutableArray<RefKind> argumentRefKinds, ImmutableArray<BoundExpression> args)
+        {
+            Debug.Assert(method.ParameterCount == args.Length);
+            return new BoundCall(
+                Syntax, receiver, method, args,
+                ImmutableArray<String>.Empty, argumentRefKinds, false, false, false,
+                ImmutableArray<int>.Empty, LookupResultKind.Viable, method.ReturnType)
```

Comment:
I think this method and the one above should pass `default(ImmutableArray<int>)` as parameter remapping